EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -157,9 +159,11 @@ The observation model is a negative binomial distribution parameterised with mea
 ```math
 \sigma_{\text{rel}} =(1/\sqrt{\mu}) + (1 / \sqrt{r}).
 ```
-It is standard to use a half-t distribution for standard deviation priors (e.g. as argued in this [paper](http://www.stat.columbia.edu/~gelman/research/published/taumain.pdf)); we specialise this to a Half-Normal prior and use an _a priori_ assumption that a typical observation fluctuation around the mean (when the mean is $\sim\mathcal{O}(10^2)$) would be 10%. This implies a standard deviation prior,
+It is standard to use a half-t distribution for standard deviation priors (e.g. as argued in this [paper](http://www.stat.columbia.edu/~gelman/research/published/taumain.pdf)); we specialise this to a Half-Normal prior and use an _a priori_ assumption that a typical observation fluctuation around the mean (when the mean is $\sim\mathcal{O}(10^2)$) would be 1%, which is close to Poisson noise.
+
+This implies a standard deviation prior,
 ```math
-1 / \sqrt{r} \sim \text{HalfNormal}\Big(0.1 ~\sqrt{{\pi \over 2}}\Big).
+1 / \sqrt{r} \sim \text{HalfNormal}\Big(0.01 ~\sqrt{{\pi \over 2}}\Big).
```

Comment:
Why not update the default?

Reply:
I was just trying out a lower variance neg bin sampling... I think deciding the appropriate defaults is an issue.